cli/index.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+var program = require('commander');
+var resize = require('./resize');
+
+program
+	.version('0.0.1')
+	.description('Image manipulation in JavaScript.');
+
+program
+	.command('resize <image> <sizes...>')
+	.alias('r')
+	.description('Resize an image')
+	.option('-c --crop', 'Hard crop an image to the specified sizes')
+	.option('-f --fill', 'Image will be stretched to specified sizes')
+	.option('-o --out-file <name>', 'Name of the output file. Size strings will be appended to this file name')
+	.option('-d --out-dir <dir>', 'Path to the output directory')
```

Comment:
Note: I haven't implemented the `--out-dir` flag yet, but the idea is that you would give `jimp` an entire directory of images and it would go through resizing all of them as per the resize instructions and then it would spit them out into the `--out-dir` directory.